diff -uNr papirus-folders.a/papirus-folders papirus-folders.b/papirus-folders
--- papirus-folders.a/papirus-folders	2020-07-23 10:43:39.800471569 +0200
+++ papirus-folders.b/papirus-folders	2020-07-23 10:45:49.885759027 +0200
@@ -117,7 +117,7 @@
 	local color=''
 	local -a colors=()
 	local -a valid_colors=("black" "blue" "bluegrey" "brown" "cyan"
-		"deeporange" "green" "grey" "indigo" "magenta" "nordic"
+		"waterleaf" "deeporange" "green" "grey" "indigo" "magenta" "nordic"
 		"orange" "pink" "purple" "red" "teal" "violet" "white" "yaru"
 		"yellow")
 
